l10n_it_dichiarazione_intento/wizard/manually_declarations.py

```diff
@@ -36,7 +36,8 @@ def confirm(self):
         if not invoice_id:
             return res
         invoice = self.env['account.invoice'].browse(invoice_id)
-        for declaration in self.declaration_ids:
-            invoice.dichiarazione_intento_ids = [
-                (4, declaration.id)]
+        if self.declaration_ids:
+            invoice.dichiarazione_intento_ids = [(6, 0, self.declaration_ids.ids)]
```

Comment:
`invoice.dichiarazione_intento_ids` non può contenere dichiarazioni arrivate da qualcosa di diverso da `self.declaration_ids`?

Reply:
Possono essere impostate dalla righe fattura alla validazione, non ho trovato un metodo per far andare in conflitto le due cose.